src/Debugger/Impl/rtvs/R/overrides.R

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for license information.
+
+view <- function(x, title) {
+  if(is.function(x) || is.data.frame(x) || is.table(x) || 
+     is.matrix(x) || is.list(x) || is.ts(x) || length(x) > 1) {
+    if (missing(title)) {
+      title <- ""
+    }
+    invisible(rtvs:::send_message('View', deparse(substitute(x)), title))
+  } else {
+    print(x)
+  }
+}
+
+browser <- function(url) {
+  rtvs:::send_message('Browser', url)
+}
+
+setwd <- function(dir) {
+  old <- .Internal(setwd(dir))
+  rtvs:::send_message('setwd', dir)
+  invisible(old)
+}
+
+library <- function(package, ...) {
+  args <- list(...)
+  if (missing(package) && length(args) == 0) {
+    invisible(rtvs:::send_message('library'))
+  } else {
+    pkgname <- as.character(substitute(package))
```

Comment:
That's really weird - passing an argument shouldn't cause it to be evaluated (since arguments are promises). 

I'm more concerned with the fact that we're calling `list(...)`. That will cause evaluation of all arguments in that list. If `library` adds some argument in the future that they expect to evaluate lazily, we will break that.